chrome/fed-modules.json

```diff
@@ -302,6 +302,15 @@
             }
         ]
     },
+    "provisioning": {
+        "manifestLocation": "/apps/provisioning/fed-mods.json",
+        "modules": [
```

Comment:
I think even the modules array can be empty if we don't need any routes. The module is still going to be accessible vial the scalprum loader. This is just for chrome to know about the remote container. But it does not have to prepare any routes.

Reply:
right! make sense, removed, thanks ! :)